addons/functions/functions/fnc_continueDiggingTrench.sqf

```diff
@@ -120,24 +111,16 @@ if (_actualProgress == 0) then {
 
     private _actualProgress = _trench getVariable ["ace_trenches_progress", 0];
     private _diggerCount = count (_trench getVariable [QGVAR(diggers),[]]);
-    private _animationPhase = _trench animationSourcePhase "rise";
-    private _diff = (_trench getVariable [QGVAR(diggingSteps), (([configFile >> "CfgVehicles" >> typeOf _trench >> QGVAR(offset), "NUMBER", 2] call CBA_fnc_getConfigEntry)/(_digTime*10))]) * _diggerCount;
-    
-    _trench animateSource ["rise", _animationPhase + _diff, true];
-    _trench setVariable ["ace_trenches_progress", _actualProgress + ((1/_digTime)/10) * _diggerCount, true];
+    private _newProgress =  _actualProgress + ((1/_digTime)) * _diggerCount;
+    _trench setVariable ["ace_trenches_progress", _newProgress, true];
```

Comment:
this line can now be removed, right

Reply:
9b92c5f14b310afac79cf8cb3480bbdb3d7a3a2d